.pnpmfile.cjs

```diff
@@ -103,6 +103,7 @@ function readPackage(pkg, context) {
       addDependencies("@celo/utils", {
         randombytes: "*",
```

Comment:
@lvndry Thanks for the PR! 👍

Could you double check if it is still necessary with the new`^3` version to add the `randombytes` and `rlp` libraries as dependencies here?

Reply:
Hello, not sure how to effectively test it so I deleted the `node_modules` in live-common and the lock file in the root directory, reinstalled dependencies
Seems like all is good 👌🏿

Reply:
> Let me know if more should be done

Cool thanks for testing! What you did looks fine to me 👌